doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)
+```
+curl -o /etc/yum.repos.d/fiware-nightly.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo
+
+```
+
+Use this config for release packages:
+```
+[fiware-release]
+name=FIWARE release repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/release
```

Comment:
I have tryied to access to http://nexus.lab.fiware.org/repository/el/7/x86_64/release and I get a 404 error.

Not sure if this is the expected behaviour (in the all repository, when I access to http://repositories.lab.fiware.org/repo/rpm/6/x86_64/ I get the list of packages...)

Reply:
See previously answer